util/configure

```diff
@@ -1234,6 +1251,7 @@ _EOC_
     $msg .= <<'_EOC_';
   --without-ngx_devel_kit_module     disable ngx_devel_kit_module
   --without-http_ssl_module          disable ngx_http_ssl_module
+  --without-stream_ssl_module        disable ngx_stream_ssl_module
 
 _EOC_
 
```

Comment:
Will you add some tests to the existing test suite to cover these changes? The `t/sanity.t` file is the right place to add such tests.